tests/integration/security/rbac/util/rbac_util.go

```diff
@@ -19,18 +19,24 @@ import (
 	"net/http"
 	"strings"
 
+	"istio.io/istio/pkg/test/echo/common/response"
 	"istio.io/istio/tests/integration/security/util/connection"
 )
 
-const (
-	DenyHTTPRespCode = "403"
-)
-
 type TestCase struct {
 	Request       connection.Checker
 	ExpectAllowed bool
 	Jwt           string
-	RejectionCode string
+}
+
+func getError(req connection.Checker, expect, actual string) error {
+	return fmt.Errorf("%s to %s:%s%s: expect %s, got: %s",
+		req.From.Config().Service,
+		req.Options.Target.Config().Service,
+		req.Options.PortName,
+		req.Options.Path,
+		expect,
+		actual)
 }
 
 // CheckRBACRequest checks if a request is successful under RBAC policies.
```

Comment:
CheckRBACRequest is different from the function name Check().

Reply:
Fixed.

@pitlv2109 I'm okay with either `Check()` or `CheckRBACRequest()`, you can change it in your following PRs.